### ISSUE TITLE
BusyIndicator: faily a11y unit test

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents#3708]
**Describe the bug**

**Isolated Example**

Example code:
```
<BusyIndicator active={activitiesListLoading} className="busyIndicator">
....
</BusyIndicator>
```
when running the a11y test, it fails with this error:
```
    Expected the HTML found at $('ui5-busyindicator,.ui5-busyindicator-busy-area') to have no violations:

    <div class="ui5-busyindicator-busy-area" tabindex="0" role="progressbar" aria-valuemin="0" aria-valuemax="100" aria-valuetext="Busy" title="Please wait" aria-labelledby="ui5wc_180-label">

    Received:

    "ARIA attributes must conform to valid values (aria-valid-attr-value)"

    Fix all of the following:
      Invalid ARIA attribute value: aria-labelledby="ui5wc_180-label"

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/3.5/aria-valid-attr-value?application=axeAPI
```

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: ~0.31.9
`@ui5/webcomponents-react` version: ^0.131.9
Operating System: Mac OS
Browser: NA

**Comment by IlianaB** [comment]
Hello @gurkirpalgill ,

Could you please isolate the issue in CodeSandbox:
https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html
so we could reproduce it and check what is the problem.

Thank you,
Iliana

**Comment by gurkirpalgill** [comment]
We did you some package updates and code refactoring. Unable to reproduce the issue now.
